src/main/webapp/app/overview/course-exercises/course-exercises.component.html

```diff
@@ -1,48 +1,96 @@
-<div class="mt-3 position-relative" *ngIf="course">
-    <div class="exercise-controls d-none d-sm-block" *ngIf="course.exercises.length > 0">
-        <div ngbDropdown placement="bottom-right" class="d-inline-block">
-            <button class="btn btn-outline-primary" id="dropdownBasic1" ngbDropdownToggle>
-                {{"arTeMiSApp.courseOverview.exerciseList.sortExercises" | translate}}
-            </button>
-            <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
-                <button class="dropdown-item" (click)="groupExercises(DUE_DATE_DESC)">
-                    <fa-icon [icon]="'sort-amount-up'"></fa-icon>
-                    {{"arTeMiSApp.courseOverview.exerciseList.newFirst" | translate}}
-                </button>
-                <button class="dropdown-item" (click)="groupExercises(DUE_DATE_ASC)">
-                    <fa-icon [icon]="'sort-amount-down'"></fa-icon>
-                    {{"arTeMiSApp.courseOverview.exerciseList.oldFirst" | translate}}
-                </button>
+<div class="row">
+    <div class="col-12 col-md-8">
+        <div class="mt-3 position-relative" *ngIf="course">
+            <div class="exercise-controls d-none d-sm-block" *ngIf="course.exercises.length > 0">
+                <div ngbDropdown placement="bottom-right" class="d-inline-block">
+                    <button class="btn btn-outline-primary" id="dropdownBasic1" ngbDropdownToggle>
+                        {{"arTeMiSApp.courseOverview.exerciseList.sortExercises" | translate}}
+                    </button>
+                    <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
+                        <button class="dropdown-item" (click)="groupExercises(DUE_DATE_DESC)">
+                            <fa-icon [icon]="'sort-amount-up'"></fa-icon>
+                            {{"arTeMiSApp.courseOverview.exerciseList.newFirst" | translate}}
+                        </button>
+                        <button class="dropdown-item" (click)="groupExercises(DUE_DATE_ASC)">
+                            <fa-icon [icon]="'sort-amount-down'"></fa-icon>
+                            {{"arTeMiSApp.courseOverview.exerciseList.oldFirst" | translate}}
+                        </button>
+                    </div>
+                </div>
+            </div>
+            <div class="exercise-row-container mb-3" *ngFor="let weekKey of weeklyIndexKeys">
+                <div class="control-label" [ngClass]="weeklyExercisesGrouped[weekKey].isCurrentWeek ? 'text-primary': ''"
```

Comment:
I think it would be clearer using an object instead of the ternary operator:

`<div class="control-label" [ngClass]="{ 'text-primary': weeklyExercisesGrouped[weekKey].isCurrentWeek}"`

https://angular.io/api/common/NgClass

Reply:
thanks is done 👍